tools/nimbus-gradle-plugin/src/main/groovy/org/mozilla/appservices/tooling/nimbus/NimbusGradlePlugin.groovy

```diff
@@ -234,7 +234,7 @@ class NimbusPlugin implements Plugin<Project> {
         if (os.contains("win")) {
             osPart = "pc-windows-gnu"
         } else if (os.contains("nix") || os.contains("nux") || os.contains("aix")) {
-            osPart = "unknown-linux-musl"
+            osPart = "unknown-linux-gnu"
```

Comment:
Given we're doing a [contains query](https://github.com/mozilla/application-services/pull/5569/files#diff-e33c27dda84611089878b9ee8717af198a7985ddb9da3d299be2bacb46a3f143R208), I think we can change this to:

```suggestion
            osPart = "unknown-linux"
```

and leaves us some flexibility around what we build, rather than it being fussy here. WDYT?

Reply:
I like it, just updated the PR with that suggestion.